### PR TITLE
Add serverKey to configuration and set version to version of nakama-js

### LINKED ISSUE #2: Missing secret key variable in ct.js client - shipping to a production env impossible
Nakama has a `server key` variable. Currently it is `defaultkey` which is bad and should not be used for production! 

Task:
- Add a field in ct.js for you to be able to change the secret key

### PR DESCRIPTION
**Pull request by ronlievens**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+root = true
```
Comment (alexandargyurov) [inline .editorconfig:1]: Please add this file to the `.gitignore` file

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
-node_modules/
-dist/
```
Comment (alexandargyurov) [inline .gitignore:2]: Why was `dist/` removed?

File: dist/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+*
```
Comment (alexandargyurov) [inline dist/.gitignore:1]: I don't think this file is needed if we have `dist/` in the original `.gitignore`

File: module.json [hunk 4]
```diff
@@ -1,12 +1,16 @@
 {
     "main": {
         "name": "ct.nakama",
-        "tagline": "Implementation of heroiclabs/nakama to enable multiplayer games",
-        "version": "0.0.2",
+        "tagline": "A wrapper for nakama-js to enable multiplayer games",
+        "version": "2.3.0",
```
Comment (alexandargyurov) [inline module.json:5]: Why `2.3.0`? I would say bump it up to `0.0.3`

File: module.json [hunk 5]
```diff
@@ -18,15 +22,22 @@
...
             "name": "Host",
             "key": "clientHost",
             "default": "127.0.0.1",
-            "type": "code"
+            "type": "text"
         },
         {
             "name": "Port",
             "key": "clientPort",
+            "help": "",
```
Comment (alexandargyurov) [inline module.json:40]: Not used / needed - let's either remove it or add in help text

File: package.json [hunk 6]
```diff
@@ -1,38 +1,38 @@
...
-    "babel-loader": "^8.2.2",
-    "eslint": "^7.20.0",
-    "jest": "^26.6.3",
-    "jest-localstorage-mock": "^2.4.7",
-    "npm-build-zip": "^1.0.3",
-    "webpack": "^5.22.0",
-    "webpack-cli": "^4.5.0"
-  }
+    "name": "ct.nakama",
+    "version": "2.3.0",
```
Comment (alexandargyurov) [inline package.json:3]: Same applies here. Let's go with `0.0.3`

**Review by alexandargyurov — CHANGES_REQUESTED** [review]
Looks good, just left some comments

File: .gitignore (hunk 2 above)
Comment (ronlievens) [reply .gitignore:2]: i added the .gitignore to the dist directory, so when checking the code out en running the code without error of missing dist folder

File: dist/.gitignore (hunk 3 above)
Comment (ronlievens) [reply dist/.gitignore:1]: this is so the directory is in git when checking thge code out

File: module.json (hunk 4 above)
Comment (ronlievens) [reply module.json:5]: the idea is to match the version number of nakama-js. (just a thought)

File: .editorconfig (hunk 1 above)
Comment (ronlievens) [reply .editorconfig:1]: this file ensures all the editors behave the same. (i can remove it)

**Pull request closed without merging**